examples.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/russross/blackfriday"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	verbose bool
+	skipped bool
+)
+
+var specsByMediaType = map[string]string{
+	"application/vnd.oci.image.manifest.v1+json":      "schema/image-manifest-schema.json",
+	"application/vnd.oci.image.manifest.list.v1+json": "schema/manifest-list-schema.json",
```

Comment:
@vbatts @philips @wking There is already quite a but of magic going on here. I thought it would sensible to have this file mapping explicitly called out. I suspect this list will remain small, so maintaining it should be of minimal burden.